chain/indexer-primitives/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "indexer-primitives"
+version = "0.0.0"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+publish = true
+# Please update rust-toolchain.toml as well when changing version here:
+rust-version = "1.56.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { version = "1", features = [ "derive" ] }
+serde_json = "1.0.55"
+
+near-primitives = { path = "../../core/primitives" }
```

Comment:
@miraclx Could you review the newly added crate config, please? (We plan to get it published to crates.io)

Reply:
Looks good to me. 👍🏽